frontend/public/components/nav/nav-header.tsx

```diff
@@ -29,7 +68,12 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({ onPerspectiveSelect
       event.preventDefault();
       if (perspective.properties.id !== activePerspective) {
         setActivePerspective(perspective.properties.id);
-        history.push(perspective.properties.getLandingPageURL(flags));
+        if (perspective.properties.id !== ACM_LINK_ID) {
+          history.push(perspective.properties.getLandingPageURL(flags));
+        } else {
+          // For ACM link, replace entire URL
+          window.location.href = perspective.properties.getLandingPageURL(flags);
+        }
```

Comment:
This is minor, but I'd flip the logic here. I find it a little harder to think in negatives.

```suggestion
        if (perspective.properties.id === ACM_LINK_ID) {
          // For ACM link, replace entire URL
          window.location.href = perspective.properties.getLandingPageURL(flags);
        } else {
          history.push(perspective.properties.getLandingPageURL(flags));
        }
```

Reply:
fixed